package.json

```diff
@@ -35,6 +35,7 @@
     "columnify": "^1.5.4",
     "fs-extra": "^9.0.1",
     "inquirer": "^7.3.0",
+    "keytar": "^6.0.1",
```

Comment:
Why do we need to add this dependency?

Reply:
i'm not sure what this is, I didn't add it manually

Reply:
Please remove this. It's an [optional dependency](https://github.com/twilio/twilio-cli-core/blob/main/package.json#L56). Thanks!